app/api/models/genome.py

```diff
@@ -14,7 +14,6 @@
 from core.config import ASSEMBLY_URLS
 
 logging.getLogger().handlers = [InterceptHandler()]
-from loguru import logger
```

Comment:
Are you removing all places where loguru is used? Do you also want to remove it from requirements.txt and pyproject.toml?

Reply:
No, I'm not. Loguru is used in logging.py